ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -1077,3 +1077,189 @@
+          error_status: Failed. Add comment /test-kind-aws-ipv6-networkpolicy to re-trigger.
+          triggered_status: null
+          started_status: null
+          wrappers:
+          - timeout:
+              fail: true
+              timeout: 135
+              type: absolute
+          publishers: []
+      - '{name}-{test_name}-for-pull-request':
+          test_name: kind-ipv6-conformance
+          node: 'antrea-aws-node'
+          description: 'This is the {test_name} test for {name}.'
+          branches:
+             - ${{sha1}}   
+          builders:
+             - builder-kind-ipv6-ds-conformance:
+                  conformance_type: 'conformance'
+                  ip_version: 'ipv6'
+          trigger_phrase: ^(?!Thanks for your PR).*/test-kind-aws-ipv6-(conformance|all).*
+          white_list_target_branches: []
+          allow_whitelist_orgs_as_admins: true
+          admin_list: '{antrea_admin_list}'
+          org_list: '{antrea_org_list}'
+          white_list: '{antrea_white_list}'
+          only_trigger_phrase: true
+          trigger_permit_all: true
+          status_context: jenkins-kind-ipv6-conformance
+          status_url: --none--
+          success_status: Build finished.
+          failure_status: Failed. Add comment /test-kind-aws-ipv6-conformance to re-trigger.
+          error_status: Failed. Add comment /test-kind-aws-ipv6-conformance to re-trigger.
+          triggered_status: null
+          started_status: null
+          wrappers:
+          - timeout:
+              fail: true
+              timeout: 135
+              type: absolute
+          publishers: []
+      - '{name}-{test_name}-for-pull-request':
+          test_name: kind-ipv6-e2e
+          node: 'antrea-aws-node'
+          description: 'This is the {test_name} test for {name}.'
+          branches:
+             - ${{sha1}}      
+          builders:
+             - builder-kind-ipv6-ds-e2e:
+                  test_type: 'e2e'
+                  ip_version: 'ipv6'
+          trigger_phrase: ^(?!Thanks for your PR).*/test-kind-aws-ipv6-(e2e|all).*
+          white_list_target_branches: []
+          allow_whitelist_orgs_as_admins: true
+          admin_list: '{antrea_admin_list}'
+          org_list: '{antrea_org_list}'
+          white_list: '{antrea_white_list}'
+          only_trigger_phrase: true
+          trigger_permit_all: true
+          status_context: jenkins-kind-ipv6-e2e
+          status_url: --none--
+          success_status: Build finished.
+          failure_status: Failed. Add comment /test-kind-aws-ipv6-e2e to re-trigger.
+          error_status: Failed. Add comment /test-kind-aws-ipv6-e2e to re-trigger.
+          triggered_status: null
+          started_status: null
+          wrappers:
+          - timeout:
+              fail: true
+              timeout: 135
+              type: absolute
+          publishers: []
+      - '{name}-{test_name}-for-pull-request':
```

Comment:
For CI testing using the public cloud, I would recommend using period job and enabling capa to save costs. Additionally, for instances using EC2, it is better to distinguish the job in the Lab Jenkins by adding 'aws' to its test name.

Reply:
@XinShuYang 
Periodic jobs are more suitable for tasks that don't need to run with every code change. Usually We might prefer to run Tests manually  in dual stack/ipv6  by trigger phrase .

Cost-saving measures like CAPA are valuable, however we will also run multiple kind jobs on a single instance in future which will optimize the instance cost.

Reply:
Implementing as a periodic job in a meaningful way is more complicated. If we want to save budget, we need to not only deploy kind cluster on demand, but also deploy VMs for kind clusters on demand, and we need at least a VM in AWS to work as a Jenkins node. I think now we deleted Nephe resources from the cloud, we have budget for 2 long-live kind VMs.